index.js

```diff
@@ -0,0 +1,26 @@
+// import express (after npm install express)
+const express = require('express');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 3000;
+
+// JSON Data
+var data  = require('./package.json');
+
+// create a route for the app
+app.get('/', (req, res) => {
+  res.send('Hello World');
+});
+
+// create a route for the /info endpoint
+app.get('/info', (req, res) => {
+    res.send("{ serverName: '" + data.name + "', serverVersion: '" + data.version + "' }");
```

Comment:
Can't we create a JSON, and then send it or stringify and send it? Did you try?

Reply:
Well, yes the task asks to return JSON, but stringifying it causes double quotes to come into play, will it be fine against single quotes required for tasks?